src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,455 @@
+        - name: Deletes the Rule
+          text: helps['sb rule delete --resource-group myresourcegroup --namespace-name mynamespace --topic-name {topicname}
+           --subscription-name {subscriptionname} --name {rulename}']
+
+    """
+
+helps['sb alias check_name_availability'] = """
+    type: command
+    short-summary: Check the availability of the Alias (Geo DR Configuration) Name 
+    examples:
+        - name: Check the availability of the Alias (Geo DR Configuration) Name
+          text: helps['sb alias check_name_availability --resource-group myresourcegroup --namespace-name primarynamespace
+           --alias myaliasname']
+
+    """
+
+helps['sb alias create'] = """
+    type: command
+    short-summary: Creats Alias (Geo DR Configuration) for the give Namespace 
+    examples:
+        - name: Creats Alias (Geo DR Configuration) for the give Namespace
+          text: helps['sb alias create  --resource-group myresourcegroup --namespace-name primarynamespace
+           --alias myaliasname --partner-namespace {id}']
+
+    """
+
+helps['sb alias show'] = """
+    type: command
+    short-summary: shows details of Alias (Geo DR Configuration) for Primay/Secondary Namespace
+    examples:
+        - name:  show details of Alias (Geo DR Configuration)  of the Primary Namespace 
+          text: helps['sb alias show  --resource-group myresourcegroup --namespace-name primarynamespace
+           --alias myaliasname']
+		- name:  Get details of Alias (Geo DR Configuration)  of the Secondary Namespace 
+          text: helps['sb alias show  --resource-group myresourcegroup --namespace-name secondarynamespace
+           --alias myaliasname']
+
+    """
+
+helps['sb alias break-pairing'] = """
+    type: command
+    short-summary: Disables the Disaster Recovery and stops replicating changes from primary to secondary namespaces
+    examples:
+        - name:  Disables the Disaster Recovery and stops replicating changes from primary to secondary namespaces 
+          text: helps['sb alias break-pairing  --resource-group myresourcegroup --namespace-name primarynamespace
+           --alias myaliasname']
+
+    """
+
+helps['sb alias fail-over'] = """
+    type: command
+    short-summary: Envokes GEO DR failover and reconfigure the alias to point to the secondary namespace
+    examples:
+        - name:  Envokes GEO DR failover and reconfigure the alias to point to the secondary namespace 
+          text: helps['sb alias fail-over  --resource-group myresourcegroup --namespace-name secondarynamespace
+           --alias myaliasname']
+
+    """
+
+helps['sb alias delete'] = """
+    type: command
+    short-summary: Delete Alias(Disaster Recovery configuration) request accepted
+    examples:
+        - name:  Delete Alias(Disaster Recovery configuration) request accepted 
+          text: helps['sb alias delete  --resource-group myresourcegroup --namespace-name secondarynamespace
+           --alias myaliasname']
+
+    """
```

Comment:
The examples in this file are not correct.
`helps['sb alias delete  --resource-group myresourcegroup --namespace-name secondarynamespace --alias myaliasname']` is not a valid command.